### PR TITLE
Fixes #292

### LINKED ISSUE #292: Cluster Slot size report inaccurate with certain policies
Plugin /20 Cluster/18 Cluster Slot Sizes.ps1 returns false positives when using resource reservations (and possibly dedicated failover hosts?) as slot sizes are not calculated.

There should be a check to determine what admission control policy is in use.

### PR DESCRIPTION
**Pull request by Sneddo**
Added type check as slots are only valid for "hosts failures the cluster tolerates" admission control policy.

### CODE REVIEW

File: Plugins/20 Cluster/18 Cluster Slot Sizes.ps1 [hunk 1]
```diff
@@ -2,21 +2,22 @@
...
-			$SlotInfo += $Details
-		}
-	}
-	$SlotCHK = @($SlotInfo | Where { $_.AvailableSlots -lt $numslots})
+   $SlotInfo = @()
+   Foreach ($Cluster in ($Clusters)){
+      If ($Cluster.ExtensionData.Configuration.DasConfig.Enabled -eq $true -and 
+          $Cluster.ExtensionData.Configuration.DasConfig.AdmissionControlPolicy.getType() -eq [VMware.Vim.ClusterFailoverLevelAdmissionControlPolicy]){
+         $SlotDetails = $Cluster.ExtensionData.RetrieveDasAdvancedRuntimeInfo()
+         $Details = "" | Select Cluster, TotalSlots, UsedSlots, AvailableSlots
```
Comment (vScripter) [inline Plugins/20 Cluster/18 Cluster Slot Sizes.ps1]: What about modifying this bit of code and casting to a proper PSObject?
```
$Details = [PSCustomObject] @{
    Cluster = $Cluster.Name
    TotalSlots = $SlotDetails.TotalSlots
    UsedSlots = $SlotDetails.UsedSlots
    AvailableSlots = $SlotDetails.UnreservedSlots
}
$SlotInfo += $Details
```

**Comment by vScripter** [comment]
Updated title; testing automatic PR <--> Issue connection in Waflle board

**Comment by vScripter** [comment]
Another title change; testing Waffle Board views and impact of utilizing [GitHub Closing Keywords](https://help.github.com/articles/closing-issues-via-commit-messages/)

File: Plugins/20 Cluster/18 Cluster Slot Sizes.ps1 (hunk 1 above)
Comment (Sneddo) [inline Plugins/20 Cluster/18 Cluster Slot Sizes.ps1]: Yeah, good idea actually.

**Pull request merged**